### ISSUE TITLE
Move donate button next to join button

### ISSUE THREAD

**Issue by ianroberts131** [publiclab/plots2#1734]
This was brought up while working through #1710, but split into a separate issue due to additional complexities.

Currently, the 'Donate' and 'Store' buttons are hidden once the screen size gets too small. In addition to moving the 'Donate' button over to the right of the 'Join' button, make it responsive so that it doesn't simply hide on smaller windows, but rather moves with the 'Join' button into the dropdown.

Does this sound right  @jywarren ? If so, I will go ahead and get started on this once #1710 is merged.

**Comment by jywarren** [comment]
Solved! Thanks!
